.github/workflows/unittests.yml

```
@@ -10,26 +10,15 @@ jobs:
   TFLite:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'ci-skip')"
-    container:
-      image: tensorflow/tensorflow:2.2.0-custom-op-ubuntu16
-
     steps:
       - uses: actions/checkout@v1
         with:
           submodules: true
```

I think this should now also allow us to use the newest version of the checkout action:
```suggestion
      - uses: actions/checkout@v2
        with:
          submodules: true
          fetch-depth: 0
```